src/handleProviderRequest.test.ts

```diff
@@ -520,15 +521,15 @@ describe('handleProviderRequest', () => {
       {
         id: 1,
         method: 'wallet_watchAsset',
-        params: {
+        params: [{
```

Comment:
will need to convert these to arrays in the app / bx if we use this rpc method directly